rpcserver.go

```diff
@@ -408,7 +408,14 @@ func (r *rpcServer) sendCoinsOnChain(paymentMap map[string]int64,
 		return nil, err
 	}
 
-	return r.server.cc.wallet.SendOutputs(outputs, feeRate)
+	tx, err := r.server.cc.wallet.SendOutputs(outputs, feeRate)
+
+	if tx == nil {
```

Comment:
Error should be checked instead?

Reply:
I think this is closest to what it was. If SendOutputs misbehaves, it could still be that tx and err are both nil.

Reply:
Since it's not possible for `SendOutputs` to return `nil, nil`, it think it's better to just check the error. Usually, changes that allow returning `nil, nil` from a function/method also add a note to its `godoc`.

Reply:
I don't think it matters, because the diff is only in return value combinations that never happen. But changed anyway to yours liking :smile: